tests/test-cluster.py

```diff
@@ -29,7 +29,7 @@
 extraCNIEnv:
   IPv4_SUPPORT: true
   IPv4_CLUSTER_CIDR: 10.3.0.0/16
-  IPv4_SERVICE_CIDR: 10.153.183.0/24
```

Comment:
Why did you change that to 152? Is there a problem with 1.53?

Reply:
It was causing problems when we enabled DNS addon. This was because the [DNS IP](https://github.com/canonical/microk8s-core-addons/blob/main/addons/dns/enable#L50) is not in the range of this cidr.

Reply:
This is a good catch. Lets create a follow up card to make the dns addon get the subnet from cni-env.